README.md

```diff
@@ -544,91 +575,86 @@ PayCart payCart = PayCart.builder()
   .build();
 ```
 
-Additionally `PayCart` provides two functions to create and prepare 
+Additionally, `PayCart` provides two functions to create and prepare:
```

Comment:
PayCart has 2 functions maskedWalletRequest and fullWalletRequest

Maybe it's better to have this:

Additionally, PayCart provides two functions to help with Masked Wallet and Full Wallet requests:

`maskedWalletRequest` to request Masked Wallet information
```java
MaskedWalletRequest maskedWalletRequest = payCart.maskedWalletRequest(ANDROID_PAY_PUBLIC_KEY);
```

`fullWalletRequest` to request Full Wallet information
FullWalletRequest fullWalletRequest = payCart.fullWalletRequest(maskedWallet);